File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    for i in babbling:
        for z in ["aya", "ye", "woo", "ma"]:
            if z in i:
                i = i.replace(z,"-")          # "aya", "ye", "woo", "ma" 가 포함된 글자를 -로 치환
        i = "".join(i.split("-"))             # 최종 검열전에 상관 없었던 -를 삭제
        if len(i) == 0:
            answer += 1
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
    int answer = 0;
    for (String i : babbling){
        if(i.contains("aya")){
            i = i.replace("aya", "-");
        }

        if(i.contains("ye")){
            i = i.replace("ye", "-");
        }

        if(i.contains("woo")){
            i = i.replace("woo", "-");
        }

        if(i.contains("ma")){
            i = i.replace("ma", "-");
        }

        i = i.replace("-", "");

        if (i.isEmpty()){
            answer += 1;
        }
        
    }
    return answer;
}
}